spec/features/user_profile_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'rails_helper'
+
+RSpec.feature 'User profile', type: :feature, js: true do
+  let!(:user) { FactoryBot.create(:user) }
+  let!(:first_event) { FactoryBot.create(:event, user: user, finish_date: '31/12/2018', start_date: '30/12/2018') }
+  let!(:second_event) { FactoryBot.create(:event, user: user, finish_date: '01/01/2019', start_date: '31/12/2018') }
+  let!(:third_event) { FactoryBot.create(:event, user: user, finish_date: '21/12/2018', start_date: '20/12/2018') }
+  let!(:target_to_first_event) { FactoryBot.create(:target, event: first_event) }
+  let!(:target_to_second_event) { FactoryBot.create(:target, event: second_event) }
+  before(:each) do
+    sign_in(user)
+    visit user_path(id: user.id)
+  end
+
+  it 'show user data' do
+    expect(page).to have_content user.first_name
+    expect(page).to have_content user.last_name
+    expect(page).to have_content user.phone
+    expect(page).to have_content user.project.name
+    expect(page).to have_content user.project.desc
+  end
+
+  it 'show sorted event' do
+    finish_date_of_events_from_page = page.all('.event-finish-date').map(&:text)
```

Comment:
Rename `finish_date_of_events_from_page` to `page_events` and move it after expected_finish_date_list